Source/SpatialGDK/Private/EngineClasses/SpatialActorChannel.cpp

```diff
@@ -139,8 +119,19 @@ void USpatialActorChannel::Close()
 	Super::Close();
 }
 
-TArray<uint16> USpatialActorChannel::SkipOverChangelistArrays(FObjectReplicator& Replicator)
+bool USpatialActorChannel::IsDynamicArrayHandle(UObject* Object, uint16 Handle)
+{
+	check(ObjectHasReplicator(Object));
+	FObjectReplicator& Replicator = FindOrCreateReplicator(Object).Get();
+	TSharedPtr<FRepLayout>& RepLayout = Replicator.RepLayout;
```

Comment:
Is the reference here by intent? it somewhat bypasses the refcounting

Reply:
Yes it is, there's no need to increase the refcount when we are just trying to read a value inside the RepLayout.